src/serializers/base.js

```diff
@@ -30,7 +30,7 @@ class Serializer {
 	 *
 	 * @memberof Serializer
 	 */
-	init(broker) {
+	init (broker) {
```

Comment:
What kind of tool did you use for formatting? Because I don't like this form that it adds extra space between method name and params.